### DISCUSSION TITLE
Resolve this compile fbt code

### DISCUSSION THREAD

**Discussion by tienlx97** [facebookincubator/fbt#410] [Q&A] [answered]
- I found that: [facebook.com]() and [workplace.com]() use `fbt` to manage i18n.
- I read this compile code: `fbt._("{label}, Selected: {date}", [fbt._param("label", e), fbt._param("date", d("datePickerUtils").maybeFormatDate(k, w))])` when I inspect websites. 
- My question is: How can I describe it as normal code (decompile it)

**Comment by jrwats** [comment, marked as answer]
```jsx
<fbt desc="...">
  <fbt:param name="label">{e}</fbt:param>,
  Selected: <fbt:param name="date">{...}</fbt:param>
</fbt>
```

Alternatively you can use the ["functional" API ](https://facebook.github.io/fbt/docs/params)
```js
fbt(fbt.param('label', e) + ', Seleced: ' + fbt.param('date', ...), "desc ...")
```

**Reply by tienlx97** [reply]
That's great ! Thank you very much

**Reply by tienlx97** [reply]
@jrwats can you help me with this compile: 

```js
function p(title, badgeCount) {
  return badgeCount > 0
    ? fbt._(
        {
          "*": "{title}, {number} new items",
          _1: "{title}, 1 new item",
        },
        [fbt._plural(badgeCount, "number"), fbt._param("title", title)]
      )
    : title;
}
```

**Reply by kayhadrin** [reply]
See reply in https://github.com/facebook/fbt/discussions/411#discussioncomment-8190328
